packages/fxa-settings/src/components/WarningMessage/index.tsx

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+type WarningMessageProps = {
+  warningMessageFtlId: string;
+  warningType: string;
+  warningMessage: string;
+};
+
+const WarningMessage = ({
+  warningMessageFtlId,
+  warningType,
+  warningMessage,
+}: WarningMessageProps) => {
+  return (
+    <div className="my-8 px-4 text-sm" data-testid="warning-message-container">
+      <FtlMsg id={warningMessageFtlId}>
```

Comment:
Non-blocking / likely not worth changing at this point, but I think I'd asked this somewhere and I don't see my comment now, what other pages do we foresee using this component? Kind of wondering if it shouldn't be a component to avoid complexity but if we're going to use it 2-3+ times then this makes sense.

It might also be nice if `warningMessage` came in as a child of the component, like,

```
<WarningMessage ftlId="whatever" warningType="whatever">
  Main message here
</WarningMessage/>
```
... but that's just because passing really long strings as a prop feels funny to me. I'd just leave it as-is for now though.

Reply:
I'll need it at least for https://github.com/mozilla/fxa/pull/14627/, not sure yet if used anywhere else